docs/source/validation_rules.rst

```diff
@@ -0,0 +1,714 @@
+
+    - ``list like``
+
+      - Assume entries are either lists or like a list but do not verify that entries are comma separated lists, and attempt to parse into a list
+
+      - Single values, or lists of length one, can be entered without a comma delimiter
+
+- Can use ``list`` rule in conjunction with ``regex`` rule to validate that the items in a list follow a specific pattern.
+
+  - All the values in the list need to follow the same pattern. This is ideal for when users need to provide a list of IDs.
+
+- Default behavior: raises ``error``
+
+Regex Validation Type
+---------------------
+
+regex
+~~~~~
+
+- Use the ``regex`` validation rule when you want to require that a user input values in a specific format, i.e. an ID that follows a particular format.
+
+- Format:
+
+  - ``regex <module> <regular_expression> <raised message level>``
+
+  - Module: is the Python ``re`` module that you want to use. A common one would be search. Refer to Python ``re`` source material to find the most appropriate module to use.
+
+  - Single spaces separate the three strings.
+
+- Example:
+
+  - ``regex search [0-9]{4}\/[0-9]*``
+
+  - The regular expression defined above allows comparison to an expected format of a histological morphology code.
+
+- Default behavior: raises ``error``
+
+- Notes/tips/warnings
+
+  - `regex101.com <https://regex101.com/>`_ is a tool that can be used to build and validate the behavior of your regular expression
+
+  - If the module specified is match for a given attribute's validation rule, regex match validation will be preformed in Google Sheets (but not Excel) real-time during metadata entry.
+
+  - The ``strict_validation parameter`` (in the config.yml file for CLI or in manifest generation REST API calls) sets whether to stop the user from entering incorrect information in a Google Sheets cell (``strict_validation = true``) or simply throws a warning (``strict_validation = false``). Default: ``true``.
+
+  - ``regex`` validation in Google Sheets is different than standard regex validation (for example, it does not support validation of digits). See `this documentation <https://github.com/google/re2/wiki/Syntax>`_ for details on Google regex syntax. It is up to the user/modeler to validate that ``regex match`` is working in their manifests, as intended. This is especially important if the ``strict_validation`` parameter is set to ``True`` as users will be blocked from entering incorrect data. If you are using Google Sheets and do not want to use real-time validation use ``regex search`` instead of ``regex match``.
+
+Type Validation Type
+--------------------
+
+- There are two parameters
+
+  - The first parameter is type and must be one of [ ``float``, ``int``, ```num```, ``str``]
+
+  - The second optional parameter is the msg level and must be one of [ ``error``, ``warning`` ], defaults to ``error``.
+
+- Examples: [ ``str``, ``str error``, ``str warning``]
```

Comment:
I like how you previously used:
Format: xxx
Example: xxx

Would it make sense for this part to follow the same structure for consistency?